packages/InlineEditing/src/Input/Input.js

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Input from "@paprika/input";
+import Editor from "../Editor";
+import * as sc from "./Input.styles";
+
+export default function InlineInput(props) {
+  const {
+    onClose,
+    onStart,
+    isEditing,
+    onChange,
+    onSubmit,
+    value,
+
+    /* eslint-disable react/prop-types */
+    rowIndex,
+    columnIndex,
+    /* eslint-enable react/prop-types */
+    ...moreProps
+  } = props;
+  const refInput = React.useRef(null);
+  const refInputEditor = React.useRef(null);
+  const [nextValue, setNextValue] = React.useState(value);
+
+  function handleBlur() {
+    onClose();
```

Comment:
Lets say the input says "old".
I click on in, and type in "new".
Then I click away from the input and its content changes to "old".
Then I click on the input and its content changes to "new".

Is that expected?

Reply:
fix though, the finally decision for interaction is not yet settle